tests/e2e/simpleevmtest.go

```diff
@@ -57,17 +56,22 @@ func evmCreate(ctx context.Context, rtc client.RuntimeClient, e evm.V1, signer s
 	return out, nil
 }
 
-func evmCall(ctx context.Context, rtc client.RuntimeClient, e evm.V1, signer signature.Signer, address []byte, value []byte, data []byte, gasPrice uint64, gasLimit uint64) ([]byte, error) {
-	tx := e.Call(address, value, data).
-		SetFeeAmount(types.NewBaseUnits(*quantity.NewFromUint64(gasPrice * gasLimit), types.NativeDenomination)).
-		GetTransaction()
+func evmCall(ctx context.Context, rtc client.RuntimeClient, e evm.V1, signer signature.Signer, address []byte, value []byte, data []byte, gasPrice uint64) ([]byte, error) {
+	txB := e.Call(address, value, data)
 
-	// Check if gas estimation works.
-	_, err := core.NewV1(rtc).EstimateGasForCaller(ctx, client.RoundLatest, testing.Dave.Address, tx)
+	// Check if ETH gas estimation works.
+	daveEthAddress := "Dce075E1C39b1ae0b75D554558b6451A226ffe00"
```

Comment:
Should we add this to test keys so you could do `testing.Dave.EthAddress` instead of decoding like this?

Reply:
Added